src/components/VirtualizedSectionList.re

```diff
@@ -235,4 +241,4 @@ external make:
     ~testID: string=?
   ) =>
   React.element =
-  "VirtualizedSectionList";
+  "VirtualizedSectionList";
```

Comment:
I suppose you haven't run `refmt` on these

Reply:
Yes. My `refmt` stopped working. I ran it on the new commit.

Reply:
(refmt should be run automatically via a transparent git hook)